docs/harmony.mess

```diff
@@ -10,9 +10,23 @@ First, you will want to inherit from ``org.shirakumo.fraf.trial.harmony:settings
 
 See ``harmony:make-simple-server`` for more information on accepted initargs.
 
-Second, you will want to pre-load sound files. The ``trial-harmony`` system provides assets and resources to encapsulate the Harmony objects you need to manage. To import sounds and simple music tracks, use the ``org.shirakumo.fraf.trial.harmony:sound`` to define your assets. If you want to use Harmony's environments system to do horizontal mixing, define the environment using the ``org.shirakumo.fraf.trial.harmony:environment`` asset.
+Second, you will want to pre-load sound files. The ``trial-harmony`` system provides assets and resources to encapsulate the Harmony objects you need to manage. To import sounds and simple music tracks, use the ``org.shirakumo.fraf.trial.harmony:sound`` to define your assets:
 
-In order to play back a sound, just access its resource and play it back:
+:: common lisp
+(define-asset (my-pool my-sound) trial-harmony:sound
+  #p"sound.mp3")
+::
+
+If you want to use Harmony's environments system to do horizontal mixing, define the environment using the ``org.shirakumo.fraf.trial.harmony:environment`` asset.
+
+Before a sound can be played, it must be "staged". It is sufficient to associate ``stage`` with any entity present in the associated scene:
```

Comment:
> It is sufficient to associate...

Is this assumption correct?

Reply:
It is correct, though there's also other options. You could subclass `scene` and use its stage method to do it, or you could use `preload` in `setup-scene`, or any number of ways. It's a question of how you want to organise the sounds.